src/BTVNanoCommissioning/workflows/ctag_Wctt_valid_sf.py

```diff
@@ -102,13 +104,18 @@ def process_shift(self, events, shift_name):
 
         isMu = False
         isEle = False
+        ### selections from Spandan
         if "WcM" in self.selMod or "semittM" in self.selMod:
             triggers = ["IsoMu27", "IsoMu24"]
             isMu = True
             dxySigcut = 1.0
             muNeEmSum = 0.7
             muonpTratioCut = 0.4
             isolepdz, isolepdxy, isolepsip3d = 0.01, 0.002, 2
+        ### remove muNeEmSum for cutbased
+        if "cutbased_WcM" == self.selMod:
+            muNeEmSum = 1.0
+            muonpTratioCut = 0.6
```

Comment:
@mondalspandan can we sync the selection? 
@adeiorio we see a problem with higher pT threshold (was 0.75 [before for shape](https://indico.cern.ch/event/1339673/contributions/5640056/attachments/2739110/4764230/BTV_commissioning_231023.pdf), and we observed some issue. Do you observed good data/MC agreement?

Reply:
IIRC the issue was with the `muNeEmSum` cut not being well calibrated. It kills off a lot of DY but can introduce an additional norm change. I am happy to remove this cut.

Reply:
Hi @adeiorio @mondalspandan thanks, so both energy fraction cut should in sync?
Then which one we should take?

Reply:
Let's keep `muNeEmSum = 1.0` since it's the safest.

Reply:
How about muonpTratioCut ? @mondalspandan

Reply:
So we should leave `muonpTratioCut = 0.4 and muNeEmSum = 1.0` @mondalspandan @adeiorio  are you agree on this?

Reply:
Hi @Ming-Yan I produced a new round of data-MC for 2023 with this selection and results are [here](https://adeiorio.web.cern.ch/adeiorio/BTV/Summer23/v2/ctag_Wc_WP_sf__24_10_16/)

Reply:
This looks ok to me, I think we should merge this 
@adeiorio @mondalspandan what do you think?

Reply:
It's of course ok with me :)